test/spmd/test_xla_distributed_checkpoint.py

```diff
@@ -340,6 +343,8 @@ def setUp(self):
     # Initialize the a minimal process group
     dist.init_process_group(
         backend='gloo', init_method='tcp://127.1:8932', world_size=1, rank=0)
```

Comment:
TIL `127.1` is [a thing](https://superuser.com/questions/614001/why-can-i-ping-127-1) :dizzy_face: 

Can you change this to `localhost`?

Reply:
Haha you don't like IP shorthand?! 😱 It's the same as `::1` but for IPv4. I guess it's more niche, I'll replace with `localhost`.